test/tests/functional/pbs_multiple_execjob_launch_hook.py

```diff
@@ -92,6 +94,18 @@ def test_multi_execjob_hook(self):
                                      ATTR_o: '/tmp/'})
         job.set_sleep_time(1)
         jid = self.server.submit(job)
-        self.mom.log_match(
+        retval = self.mom.log_match(
             "Modified progname value got updated from launch1",
             max_attempts=3, interval=3)
+        self.assertTrue(retval)
+
+        hook1 = ("execjob_hook1", "execjob_hook2")
+
+        for hk in hook1:
+            msg = "environment var from " + hk
+            rv = self.mom.log_match(msg, starttime=self.server.ctime,
+                                    max_attempts=10)
+            self.assertTrue(rv)
```

Comment:
You don't need to check the return value of log_match()

Reply:
Fixed.